CHANGES.txt

```diff
@@ -33,6 +33,10 @@ Changes
 
  - Changed the Enterprise License missing check message, as well as lowering
    the severity from HIGH to LOW.
+   
+  - Introduce support for single row subselects in ANY, e.g.::
+
+      Select id from t where id = ANY(Select id from t2)
```

Comment:
I wanted to provide a more real-world example, than `select 1 where 1 = any (select 1)`.